### DISCUSSION TITLE
请问如何把图的子标题放在左边呢

### DISCUSSION THREAD

**Discussion by Jamie-obj** [note286/xduts#125] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/65389500/236187161-2941180d-21cb-447a-b6f6-faf45cb86d3b.png)
如图所示，因为图片排版纵向很长，横向放图会好一点，查了一圈没找到该如何横向排列的方式

**Comment by note286** [comment, marked as answer]
你是要旋转页面吗？不建议这么做。

你可以纵向排列，参考 https://github.com/note286/xduts/discussions/40#discussioncomment-4970710 。

**Reply by Jamie-obj** [reply]
好的~
